Filter/filter.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Fontawesome Icons -->
+    <script
+      src="https://kit.fontawesome.com/dd0d05bba7.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      rel="stylesheet"
+      href="https://pastel-ui.netlify.app/css/components.css"
+    />
+
+    <!-- My Stylesheet -->
+    <link rel="stylesheet" href="/css/main.css" />
+    <link rel="stylesheet" href="/Filter/filter.css" />
+    <title>E-Commerce | Filter Component</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="navigation-brand"><a href="/index.html">MySite</a></div>
```

Comment:
The font size of this could be increased. It looks a bit tiny as of now and as that is your product branding, it should be visible properly as well as take the attention of users when they look at the navbar.

Reply:
Yes, I will be fixing the Navbar very soon. It's just temporary for now.